srcpkgs/rmpc/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'rmpc'
+pkgname=rmpc
+version=0.8.0
+revision=1
+build_style=cargo
+depends="mpd"
+short_desc="TUI MPD client with album art support"
+maintainer="Will Reed <wreed@disroot.org>"
+license="BSD-3-Clause"
+homepage="https://mierak.github.io/rmpc"
+changelog="https://raw.githubusercontent.com/mierak/rmpc/refs/heads/master/CHANGELOG.md"
+distfiles="https://github.com/mierak/rmpc/archive/refs/tags/v${version}.tar.gz"
+checksum="f2ff534ad662d6bb033f1715c27406718eb7025655f5c30c94a6b051e3a1c371"
+
+do_install() {
+	vbin "target/${RUST_TARGET}/release/rmpc"
```

Comment:
Please change `do_install` to `post_install` and let the cargo build style install the binary itself.